README.md

```diff
@@ -1,9 +1,69 @@
 # Polymer Linter
 
-Catch errors in your Polymer project before even running your code.
+Analyze your Polymer project for errors.
 
-## Work in progress
+## Usage
 
-Polymer Linter is being developed to replace [polylint][1], but is not yet complete.  For a working linter, use [polylint][1].
+The linter can be run in one of two ways: from the command line, or through our editor plugins.
+
+First though, it needs to be configured. Create a `polymer.json` file at the root of your project and ensure it has a "lint" field like so:
+
+```json
+{
+  "lint": {
+    "rules": ["polymer-2-hybrid"]
+  }
+}
+```
+
+You can specify either individual rules, or rule collections. See `polymer lint --help` for a full listing, or see [collections.ts](src/collections.ts) for details.
+
+### From the command line
+
+polymer-lint is run on the command line through Polymer CLI:
+
+```
+  npm install -g polymer-cli@next
+  cd my-project
+  polymer lint
+```
+
+By default this produces output with the exact location of any issues clearly underlined:
+
+```
+    <paper-button opan="{{nonono}}"></paper-button>
+                  ~~~~
+
+my-elem.html(9,24) warning [set-unknown-attribute] - paper-button elements do not have a property named opan. Consider instead:  open
+```
+
+### Inside your editor
+
+polymer-lint is also integrated into a number of editor plugins for instant as-you-type linting. See [the polymer editor service](https://github.com/Polymer/polymer-editor-service) for details.
+
+![Animated gif of the VSCode with the linter running.](https://cloud.githubusercontent.com/assets/1659/23933285/ad63eb62-08fa-11e7-819b-641bf83cf9c6.gif)
+
+## Writing your own lint rule
```

Comment:
Let's change the top heading of this section to something like "Extending and Contributing"